src/main/java/io/github/redouane59/twitter/helpers/AbstractRequestHelper.java

```diff
@@ -42,7 +43,7 @@ protected AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aS
   }
 
   public static void logApiError(String method, String url, String stringResponse, int code) {
-    LOGGER.error("(" + method + ") Error calling " + url + " " + stringResponse + " - " + code);
+    LOGGER.error("({}}) Error calling {} {} - {}", method, url, stringResponse, code);
```

Comment:
is it normal to have one opening and two closing ?

Reply:
Thanks, fixed